support/ebpf/frametypes.h

```diff
@@ -36,9 +32,10 @@
 // Indicates a Go frame
 #define FRAME_MARKER_GO      0xB
 
-// Indicates a frame containing information about a critical unwinding error
-// that caused further unwinding to be aborted.
-#define FRAME_MARKER_ABORT (0x7F | FRAME_MARKER_ERROR_BIT)
+// Frame flags
+#define FRAME_FLAG_ERROR          1
+#define FRAME_FLAG_RETURN_ADDRESS 2
+#define FRAME_FLAG_PID_SPECIFIC   4
```

Comment:
This `FRAME_FLAG_PID_SPECIFIC` could be omitted, and be moved back to a frame type switch in the Go code if preferred.